ksql-functional-tests/src/test/resources/query-validation-tests/simple-struct.json

```diff
@@ -590,7 +590,7 @@
           "topic": "S1",
           "timestamp": 0,
           "value": {
-            "ITEMID": "{\"CATEGORY\":{\"ID\":2,\"NAME\":\"Food\"},\"ITEMID\":6,\"NAME\":\"Item_6\"}",
+            "ITEMID": "{\"ITEMID\":6,\"CATEGORY\":{\"ID\":2,\"NAME\":\"Food\"},\"NAME\":\"Item_6\"}",
```

Comment:
Does this open the possibility of flaky test failures, if we no longer guarantee the order of these fields?

Reply:
I think it is deterministic, just not guaranteed.